addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -1413,6 +1413,14 @@ export class OdooEditor extends EventTarget {
     // ===============
 
     deleteRange(sel) {
+        // Remove all FEFF text nodes
+        for (const node of descendants(this.editable)) {
+            if (node.nodeType === Node.TEXT_NODE && node.textContent === '\uFEFF') {
+                const restore = prepareUpdate(...leftPos(node));
+                node.remove();
+                restore(); // Make sure to make <br>s visible if needed.
+            }
+        }
```

Comment:
I'm not sure it's overkill to do so because, outside the fact that if feels wrong conceptually to make changes in the whole editor just because I delete a range _somewhere_, it will creates useless mutations, that are going to be sent on the network for synchronizing between users etc. 

I think using the `commonAncestorContainer` of the range for the call to `descendants` would be a lot more efficient in that regard, in addition to be conceptually more correct.

Reply:
Ah yes indeed I hadn't thought of the collaboration! Maybe better `closestBlock` of the `commonAncestorContainer` then to account for cases like `<li>//feff//<a>//feff//[abc]//feff//</a>//feff//</li>`? That way we remove the feff _around_ the link too.